### PR TITLE
feat: add lolz

### PR DESCRIPTION
**Pull request by ravi-tomato**
Hi I would like to add lolz to the codebase

### CODE REVIEW

**Comment by JUHYUNSAN** [comment]
HFHFH

File: src/App.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ function App() {
   return (
     <div className="App">
     firstdd
+    lolz
```
Comment (JUHYUNSAN) [inline src/App.js:7]: WHY

File: src/App.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ function App() {
   return (
     <div className="App">
     firstdd
```
Comment (JUHYUNSAN) [inline src/App.js:6]: PLS CHANGE THIS

**Review by JUHYUNSAN — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 2 above)
Comment (ravi-tomato) [reply src/App.js:6]: Done

**Pull request merged**